docs/data-sources/elb_ipgroups.md

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: Dedicated Load Balance (Dedicated ELB)
+---
+
+# huaweicloud_elb_ipgroups
+
+Use this data source to get the list of ELB ipgroups.
```

Comment:
please replace all **ipgroups** to **IP groups** in the docs, logs and error message.